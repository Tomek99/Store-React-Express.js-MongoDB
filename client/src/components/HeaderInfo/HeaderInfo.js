import React from "react";
import styles from "./HeaderInfo.module.scss";
import PropTypes from "prop-types";

function HeaderInfo({ title }) {
  return <header className={styles.HeaderInfo}>{title}</header>;
}

HeaderInfo.propTypes = {
  title: PropTypes.string,
};
export default HeaderInfo;
