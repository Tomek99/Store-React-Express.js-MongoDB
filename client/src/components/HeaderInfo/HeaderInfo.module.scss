.HeaderInfo {
  font-size: 2.5rem;
  font-weight: bold;
  color: var(--text-color);
}
