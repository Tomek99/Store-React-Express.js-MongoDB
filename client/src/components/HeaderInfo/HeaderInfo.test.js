import { render } from "@testing-library/react";
import HeaderInfo from "./HeaderInfo";

test("should render HeaderInfo component", () => {
  const { container } = render(<HeaderInfo title="TEST" />);
  expect(container).toMatchSnapshot();
});
