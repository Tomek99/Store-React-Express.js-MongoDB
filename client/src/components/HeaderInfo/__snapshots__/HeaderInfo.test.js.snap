// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render HeaderInfo component 1`] = `
<div>
  <header
    class="HeaderInfo"
  >
    TEST
  </header>
</div>
`;
